docker/entry.sh

```diff
@@ -17,7 +17,7 @@ pid=0
 database_dir=/usr/share/GeoIP
 log_dir="/tmp/geoipupdate"
 log_file="$log_dir/.healthcheck"
-flags="--output"
+flags="--output -f /dev/null"
```

Comment:
Ah sorry. `GEOIPUPDATE_CONF_FILE=""` will remove the need for a configuration file.

Reply:
Are you saying I should set that in this script rather than use the flag?

Reply:
I suppose either works just as well, but yeah it'd probably be more consistent to use `GEOIPUPDATE_CONF_FILE=""`

Reply:
OK, I changed that and I also fixed the setting of `GEOIPUPDATE_DB_DIR`.